tag_manager/custom_popup_examples.rst

```diff
@@ -0,0 +1,50 @@
+.. highlight:: js
+.. default-domain:: js
+
+.. versionadded:: 10.1
+
+Custom popup template implementation examples
+=============================================
+
+Introduction
+------------
+Since version 10.1 of PPAS there is a possiblity of creating a `Custom popup` tag template. We decided to share some example implementations that can be further modified and expanded.
```

Comment:
This is too brief I think. There should be some instructions about how to add Custom Popup tag.